### PR TITLE
Script to update webapps to use fluent-bit image in ECR

### PR DESCRIPTION
**Pull request by xoen**
### Why
This is to reduce the number of pull requests we're making to DockerHub
which is causing problems because of the newly introduced rate limites.

I also ran this through https://www.shellcheck.net/

Thanks to @damacus for the help with the Bash function.

### Ticket
https://trello.com/c/Izxt87HF

### CODE REVIEW

File: scripts/update-webapps-fluent-bit-image-to-ECR.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+DEPLOYMENTS=$(kubectl -n ${NAMESPACE} get deployment | grep \\-webapp | cut -d" " -f1)
+
+for deployment in $DEPLOYMENTS; do
+    echo
+    echo "Processing '${deployment}'..."
+
+    if uses_old_image "${deployment}"; then
+        # kubectl -n "${NAMESPACE}" set image "deployment/${deployment}" ${CONTAINER}=${NEW_IMAGE}
```
Comment (andylightfoot) [inline scripts/update-webapps-fluent-bit-image-to-ECR.sh:25]: Is this meant to be commented out?

Comment (xoen) [reply scripts/update-webapps-fluent-bit-image-to-ECR.sh:25]: Nope.

Comment (andylightfoot) [reply scripts/update-webapps-fluent-bit-image-to-ECR.sh]: ok

**Review by andylightfoot — APPROVED** [review]

File: scripts/update-webapps-fluent-bit-image-to-ECR.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    kubectl -n ${NAMESPACE} get deployment "${deployment}" -o yaml | grep -q "image: ${OLD_IMAGE}"
+}
+
+
+DEPLOYMENTS=$(kubectl -n ${NAMESPACE} get deployment | grep \\-webapp | cut -d" " -f1)
+
+for deployment in $DEPLOYMENTS; do
+    echo
+    echo "Processing '${deployment}'..."
```
Comment (damacus) [inline scripts/update-webapps-fluent-bit-image-to-ECR.sh:22]:
```suggestion
    echo "\nProcessing '${deployment}'..."
```

This is slightly more "Bash Like"  :) 

But LGTM!

**Review by damacus — APPROVED** [review]

File: scripts/update-webapps-fluent-bit-image-to-ECR.sh (hunk 2 above)
Comment (xoen) [reply scripts/update-webapps-fluent-bit-image-to-ECR.sh:21]: Unfortunately don't think `echo` interprets those `\n` as you'd normally expect (I knew that but I re-tried and it didn't work, it just prints `\n` 😬 ).

**Comment by xoen** [comment]
FYI: I've run this in `alpha`. I've added [a comment](https://github.com/ministryofjustice/analytics-platform-ops/pull/373/files#diff-9cd8fa57b85af69e1e428d356094e6f18ff38fa1d0bde8e3464c8faff0e12620R16) to the script to show how you can use it to update only a whitelist of deployments, e.g. to run things in batches like I did. Merging this.

**Pull request merged**